qip-0015.md

```diff
@@ -0,0 +1,59 @@
+```
+  QIP: 15
+  Layer: Consensus (hard fork) 
+  Title: Qi UTXO Set Trimming
+  Author: jdowning100
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/quai-network/qips/wiki/Comments:QIP-00015
+  Status: Active
+  Type: Standards Track
+  Created: 2024-09-06
+  License: BSD-2-Clause
```

Comment:
we should note that this replaces qip-0012

Reply:
Done